pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1232,6 +1232,7 @@ public void openLedgerComplete(ManagedLedger ledger, Object ctx) {
                                     preCreateSubForCompaction = ((SystemTopic) persistentTopic)
                                             .preCreateSubForCompactionIfNeeded();
                                 }
+                                managedLedgerConfig.setCreateIfMissing(true);
```

Comment:
Maybe set `createIfMissing` as `true` directly is not reasonable, this will break the service configuration?

Reply:
This may depend on the original design intention of getTopicIfExists. Calling getTopicIfExists will directly set createIfMissing to false, which will affect the configuration. I don't know if this is intentional or ignored.

Reply:
However, I did make a mistake and did not consider its initial configuration. I think I should keep the configuration first, and then restore the configuration

Reply:
Thank you for pointing out.
Can you see if there are any problems now?

Reply:
Sorry, Brother Ran. The above is my ill-conceived.
I have another discussion with congbo. 
After this topic has been created, the field will  have no meaning. And each subscription of transaction needs createIfMsiing=true to create PendingAck. So it is also feasible to set it to true directly in Completely